src/types/Story.ts

```diff
@@ -24,7 +24,7 @@ export interface StoryRef {
      * @see status
      * @see Story.Status
      */
-    lifecycle_status: Story.LifecycleStatus;
+    lifecycle_status: Story.Status;
```

Comment:
The build on `master` and this branch was failing. I guess the way we used to export it doesn't work in recent versions of TypeScript or something around our build pipeline:

```tsx
// With TS namespace, doesn't work ❌
export namespace Story {
  export import LifecycleStatus = Status;
};

// Without TS namespace, works ✔️
export import LifecycleStatus = Status;
```

This was a deprecated API so just removed it. I believe it's safe, right?

Reply:
> The build on master and this branch was failing. I guess the way we used to export it doesn't work in recent versions of TypeScript or something around our build pipeline

Hm... Might be a change in the newer TS versions. 

A few thoughts: 

1. There are more of `export import` places in this codebase:
   ![image](https://github.com/user-attachments/assets/5417cea1-b62d-4bea-b425-3a6d317f7f6a)

2. Can you please try to use `export const T = X; export type T = X;` instead? Will it work?

3. If we're going to remove (which I don't mind), it has to be a major release because of the BC break.

Reply:
> 2. Can you please try to use `export const T = X; export type T = X;` instead? Will it work?

Works! 👍 🎉